README.md

```diff
@@ -1 +1,207 @@
+This is the template for Project 3 repositories. It includes all the assignment instructions and documentation to help you use GitHub.
+
+<h2> Table of Contents </h2>
+
+- [Getting Started with the Project](#getting-started-with-the-project)
+	- [Step 1: Create Your Project Repository](#step-1-create-your-project-repository)
+	- [Step 2: Repository Name and Visibility](#step-2-repository-name-and-visibility)
+	- [Step 3: Collaboration and Team Members](#step-3-collaboration-and-team-members)
+	- [Step 4: Development](#step-4-development)
+		- [Issues:](#issues)
+		- [Branches](#branches)
+		- [Commits](#commits)
+			- [Guidelines for good commit messages](#guidelines-for-good-commit-messages)
+		- [Pull Requests](#pull-requests)
+- [README, Markdown, and Report Submission](#readme-markdown-and-report-submission)
+	- [Important Elements](#important-elements)
+	- [Important Note on ToC](#important-note-on-toc)
+- [Additional Resources and Documentation:](#additional-resources-and-documentation)
+	- [General Documentation](#general-documentation)
+	- [Additional Specific Documentation](#additional-specific-documentation)
+		- [Quickstart](#quickstart)
+		- [Repositories](#repositories)
+		- [Branches](#branches-1)
+		- [Issues](#issues-1)
+		- [Bonus Documentation](#bonus-documentation)
+
+# Documentation and resources on how to use GitHub for Project 3
+## Getting Started with the Project
+
+This is a group project. As such, you're expected to equally contribute to all parts of the project. To make the collaboration easier for you, we have moved this project entirely to GitHub. However, to fully leverage the features that GitHub offers, you have to be familiar with what they are and how to use them.
+
+The goal of this project is to get you familiar with the creative process of software development for a general purpose use and how to use data structures or algorithms for those applications. To go with the professional development that the project offers in terms of general use application, you will also get more familiar with version control and GitHub platform to facilitate the collaborative work with extensive planning and organization abilities.
+
+This step by step process if linked in this video:
+- [YouTube](https://youtu.be/CaGUAi6jnik) - Doesn't have chapters but has subtitles
+- [Loom](https://www.loom.com/share/5f06c23105cf449a8f526b997cb3768b?sid=e30f822f-11c1-4ce7-8821-008932fd930a) - Has chapters and transcript
+
+### Step 1: Create Your Project Repository
+
+This is a template repository. If you want to know more about those, feel free to follow [this link](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository) to the GitHub documenation about templates and how to create them. You've used templates before for projects 1 and 2, and you will use it now, too. 
+
+The first step of your project is to create the project repository that will host your source code and documentation.
```

Comment:
Add: "**Only one member of the team needs to create a repo and everyone else copies that repo using git clone**".